<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>914</width>
    <height>644</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Welcome</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="styleSheet">
    <string notr="true">background-color: rgb(25, 25, 25);
background-color: rgb(58, 58, 58);</string>
   </property>
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="0" column="0">
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_12">
          <item>
           <layout class="QFormLayout" name="formLayout">
            <item row="0" column="1">
             <widget class="QLabel" name="label_2">
              <property name="font">
               <font>
                <family>Arial</family>
                <pointsize>26</pointsize>
                <weight>75</weight>
                <bold>true</bold>
               </font>
              </property>
              <property name="styleSheet">
               <string notr="true">color: rgb(255, 255, 255);</string>
              </property>
              <property name="text">
               <string>Max PC</string>
              </property>
             </widget>
            </item>
            <item row="0" column="0">
             <widget class="QLabel" name="label">
              <property name="text">
               <string/>
              </property>
              <property name="pixmap">
               <pixmap>100x100.png</pixmap>
              </property>
              <property name="scaledContents">
               <bool>false</bool>
              </property>
              <property name="alignment">
               <set>Qt::AlignJustify|Qt::AlignVCenter</set>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_4"/>
          </item>
          <item>
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_5">
            <item>
             <widget class="QLabel" name="lbldatetime">
              <property name="font">
               <font>
                <family>Arial</family>
                <pointsize>14</pointsize>
                <weight>75</weight>
                <bold>true</bold>
               </font>
              </property>
              <property name="styleSheet">
               <string notr="true">color: rgb(255, 255, 255);</string>
              </property>
              <property name="text">
               <string>Date and Time</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLCDNumber" name="lcdDT">
              <property name="digitCount">
               <number>15</number>
              </property>
              <property name="segmentStyle">
               <enum>QLCDNumber::Outline</enum>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QLabel" name="txtCrntUsr">
          <property name="font">
           <font>
            <family>Arial</family>
            <pointsize>12</pointsize>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">color: rgb(255, 255, 255);</string>
          </property>
          <property name="text">
           <string>Welcome, User</string>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QGroupBox" name="groupBox_2">
            <property name="styleSheet">
             <string notr="true">color: rgb(255, 255, 255);</string>
            </property>
            <property name="title">
             <string>Categorized by:</string>
            </property>
            <layout class="QHBoxLayout" name="horizontalLayout_5">
             <item>
              <widget class="QRadioButton" name="btnBrNew">
               <property name="text">
                <string>Brand New</string>
               </property>
               <attribute name="buttonGroup">
                <string notr="true">CatSelect</string>
               </attribute>
              </widget>
             </item>
             <item>
              <widget class="QRadioButton" name="btnSecH">
               <property name="text">
                <string>2nd Hand</string>
               </property>
               <attribute name="buttonGroup">
                <string notr="true">CatSelect</string>
               </attribute>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="QGroupBox" name="groupBox">
            <property name="styleSheet">
             <string notr="true">color: rgb(255, 255, 255);</string>
            </property>
            <property name="title">
             <string>Sort by</string>
            </property>
            <layout class="QHBoxLayout" name="horizontalLayout_4">
             <item>
              <widget class="QRadioButton" name="btnSeeAll">
               <property name="styleSheet">
                <string notr="true">color: rgb(255, 255, 255);</string>
               </property>
               <property name="text">
                <string>See All</string>
               </property>
               <attribute name="buttonGroup">
                <string notr="true">SortSelector</string>
               </attribute>
              </widget>
             </item>
             <item>
              <widget class="QRadioButton" name="btnCat">
               <property name="styleSheet">
                <string notr="true">color: rgb(255, 255, 255);</string>
               </property>
               <property name="text">
                <string>By Category</string>
               </property>
               <attribute name="buttonGroup">
                <string notr="true">SortSelector</string>
               </attribute>
              </widget>
             </item>
             <item>
              <widget class="QComboBox" name="cmbCat">
               <property name="styleSheet">
                <string notr="true">color: rgb(255, 255, 255);</string>
               </property>
               <item>
                <property name="text">
                 <string>Sample Category 1</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>Sample Category 2</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>Sample Category 3</string>
                </property>
               </item>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <layout class="QFormLayout" name="formLayout_2">
            <item row="0" column="1">
             <widget class="QLineEdit" name="txtSearch">
              <property name="styleSheet">
               <string notr="true">background-color: rgb(255, 255, 255);</string>
              </property>
              <property name="placeholderText">
               <string>Search Product ID or Name</string>
              </property>
             </widget>
            </item>
            <item row="1" column="1">
             <widget class="QPushButton" name="btnSearch">
              <property name="styleSheet">
               <string notr="true">background-color: rgb(69, 0, 207);
color: rgb(255, 255, 255);</string>
              </property>
              <property name="text">
               <string>Search</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QTableView" name="tblData">
            <property name="styleSheet">
             <string notr="true">
background-color: rgb(255, 255, 255);</string>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_3">
            <item>
             <widget class="QPushButton" name="btnAdd">
              <property name="styleSheet">
               <string notr="true">background-color: rgb(253, 251, 132);</string>
              </property>
              <property name="text">
               <string>Add New Item</string>
              </property>
              <property name="icon">
               <iconset>
                <normaloff>add.png</normaloff>add.png</iconset>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="btnEdit">
              <property name="styleSheet">
               <string notr="true">background-color: rgb(253, 251, 132);</string>
              </property>
              <property name="text">
               <string>Edit Item</string>
              </property>
              <property name="icon">
               <iconset>
                <normaloff>edit.png</normaloff>edit.png</iconset>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="btnRestock">
              <property name="styleSheet">
               <string notr="true">background-color: rgb(253, 251, 132);</string>
              </property>
              <property name="text">
               <string>Restock Item</string>
              </property>
              <property name="icon">
               <iconset>
                <normaloff>restock.png</normaloff>restock.png</iconset>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="btnRemove">
              <property name="styleSheet">
               <string notr="true">background-color: rgb(140, 0, 0);
color: rgb(255, 255, 255);</string>
              </property>
              <property name="text">
               <string>Remove Item</string>
              </property>
              <property name="icon">
               <iconset>
                <normaloff>delete_forever1.png</normaloff>delete_forever1.png</iconset>
              </property>
             </widget>
            </item>
            <item>
             <layout class="QFormLayout" name="formLayout_3">
              <item row="0" column="0">
               <widget class="QLabel" name="label_4">
                <property name="font">
                 <font>
                  <family>Arial</family>
                  <pointsize>14</pointsize>
                 </font>
                </property>
                <property name="styleSheet">
                 <string notr="true">color: rgb(255, 255, 255);</string>
                </property>
                <property name="text">
                 <string>Selected Items:</string>
                </property>
               </widget>
              </item>
              <item row="1" column="0" colspan="2">
               <widget class="QPlainTextEdit" name="txtSelect">
                <property name="styleSheet">
                 <string notr="true">background-color: rgb(255, 255, 255);</string>
                </property>
                <property name="readOnly">
                 <bool>true</bool>
                </property>
                <property name="placeholderText">
                 <string>Selected items appear here...</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <widget class="QPushButton" name="btnSell">
            <property name="styleSheet">
             <string notr="true">background-color: rgb(0, 84, 0);
color: rgb(255, 255, 255);</string>
            </property>
            <property name="text">
             <string>Sell Item(s)</string>
            </property>
            <property name="icon">
             <iconset>
              <normaloff>checkout.png</normaloff>checkout.png</iconset>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_7">
            <item>
             <widget class="QPushButton" name="btnAddSel">
              <property name="styleSheet">
               <string notr="true">background-color: rgb(253, 251, 132);</string>
              </property>
              <property name="text">
               <string>Add to Selection</string>
              </property>
              <property name="icon">
               <iconset>
                <normaloff>Select.png</normaloff>Select.png</iconset>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="btnClrSel">
              <property name="styleSheet">
               <string notr="true">background-color: rgb(79, 0, 0);
color: rgb(255, 255, 255);</string>
              </property>
              <property name="text">
               <string>Clear Selections</string>
              </property>
              <property name="icon">
               <iconset>
                <normaloff>close.png</normaloff>close.png</iconset>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_6">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_14">
            <item>
             <widget class="QPushButton" name="btnCustR">
              <property name="styleSheet">
               <string notr="true">background-color: rgb(253, 251, 132);</string>
              </property>
              <property name="text">
               <string>Customer Records</string>
              </property>
              <property name="icon">
               <iconset>
                <normaloff>person.png</normaloff>person.png</iconset>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="btnViewL">
              <property name="styleSheet">
               <string notr="true">background-color: rgb(253, 251, 132);</string>
              </property>
              <property name="text">
               <string>View Logs</string>
              </property>
              <property name="icon">
               <iconset>
                <normaloff>inventory.png</normaloff>inventory.png</iconset>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="pushButton">
              <property name="styleSheet">
               <string notr="true">color: rgb(0, 0, 0);
background-color: rgb(253, 251, 132);</string>
              </property>
              <property name="text">
               <string>Category Editor</string>
              </property>
              <property name="icon">
               <iconset>
                <normaloff>category.png</normaloff>category.png</iconset>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QPushButton" name="btnLogOut">
            <property name="styleSheet">
             <string notr="true">background-color: rgb(140, 0, 0);
color: rgb(255, 255, 255);</string>
            </property>
            <property name="text">
             <string>Log Out</string>
            </property>
            <property name="icon">
             <iconset>
              <normaloff>logout1.png</normaloff>logout1.png</iconset>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item row="0" column="2">
     <widget class="QLabel" name="label_7">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>914</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
 <buttongroups>
  <buttongroup name="SortSelector"/>
  <buttongroup name="CatSelect"/>
 </buttongroups>
</ui>
